src/main_screen.hpp

```diff
@@ -1,12 +1,12 @@
 // SPDX-License-Identifier: GPL-3.0-only
 #pragma once
+#include "app_state.hpp"
 #include <SFML/Graphics.hpp>
 #include <memory>
-#include "app_state.hpp"
 
 class MainScreen : public AppState {
 public:
-    MainScreen(sf::RenderWindow& window);
+    MainScreen(sf::RenderWindow &window);
```

Comment:
bruhhhhhhh 🥲
it's `Type& var_name` not `Type &var_name` 
fight me

Reply:
It's subjective. I don't mind either way. I'll change it.